ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -222,12 +253,43 @@ class ShyHeaderView: UIView, TokenizedControlInternal {
 
     private func initContentStackView() {
         contentStackView.isLayoutMarginsRelativeArrangement = true
+        contentStackView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(contentStackView)
-        contentStackView.fitIntoSuperview(usingConstraints: true)
+
+        if let wideContentStackViewTopAnchorConstraint {
```

Comment:
nit: Might be worth adding a comment here explaining why we are deactivating this constraints we the shy header's content is being initialized.

Reply:
Maybe the name `contentStackView` is a little confusing? This stack view only contains the accessoryView, and the new `wideAccessoryView` is hosted in another stack view `wideContentStackView`. When we don't have an `accessoryView` we set the top anchor constraint of the `wideContentStackView` to be equal to the current view's top anchor. But when `accessoryView` is added here(whenever a `accessoryView` is added we call `initContentStackView()`) we need to deactivate that constraints and instead constraint the top anchor of the `wideContentStackView` to be equal to the bottom anchor of the `contentStackView`. Does that make sense?

Reply:
Added a comment.